packages/builder/src/components/integration/RestQueryViewer.svelte

```diff
@@ -196,8 +196,36 @@
     }
   }
 
+  const validateQuery = async () => {
+    const forbiddenBindings = /{{\s?user\s?}}/g
```

Comment:
This will catch `{{ user }}` by itself, but won't catch things like `{{ user.email }}`. Just wanted to check that this was acceptable. Worth checking that something like `{{ user.email }}` doesn't also throw an error if used.

Reply:
Yup good catch, updated this to handle property access too. It didn't throw any errors but seems improper to leave it possible anyway I guess. A user can still technically subscript `user` I guess, dunno if that's possible in handlebars.